vizro-core/src/vizro/models/_components/form/_form_utils.py

```diff
@@ -7,7 +7,9 @@
 from vizro.models.types import MultiValueType, OptionsType, SingleValueType
 
 
-def get_options_and_default(options: OptionsType, multi: bool = False):
+def get_options_and_default(
+    options: OptionsType, multi: bool = False
+) -> tuple[OptionsType, Union[SingleValueType, MultiValueType]]:
```

Comment:
Sorry for the wrong suggestion earlier, but I think that this should be the return type. Let's try and see is there any `mypy` errors.
```suggestion
) -> tuple[OptionsType, SingleValueType]:
```